### PR TITLE
[FPBuiltinFnSelection] Handle sqrt fpbuiltins for X86 targets.

### PR DESCRIPTION
**Pull request by FreddyLeaf**
For fpbuiltin.sqrt, it should always use the native operation for
X86-based targets because the native instruction is faster (even faster
than the low-accuracy SVML implementation).

### CODE REVIEW

File: llvm/test/CodeGen/X86/fp-builtin-intrinsics-svml-sqrt.ll [hunk 1]
```diff
@@ -0,0 +1,278 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
```
[the remark this answers is not in the document]
Comment (FreddyLeaf) [reply llvm/test/CodeGen/X86/fp-builtin-intrinsics-svml-sqrt.ll:1]: Agree.

**Comment by FreddyLeaf** [comment]
> LGTM. Thanks

Thanks review! Would you also help merge? I don't have write access.

**Comment by maksimsab** [comment]
Why don't we send this patch directly into llvm-project?

**Comment by FreddyLeaf** [comment]
llvm.org doesn't have llvm/lib/CodeGen/FPBuiltinFnSelection.cpp and relate -ffp-accuracy supports.

**Comment by dm-vodopyanov** [comment]
> > LGTM. Thanks
> 
> Thanks review! Would you also help merge? I don't have write access.

@FreddyLeaf, hi, we have the following process for merging: https://github.com/intel/llvm/blob/sycl/CONTRIBUTING.md#merge:
```
... Feel free to ping @intel/llvm-gatekeepers if your PR is green and can be merged. ...
```
The PR must also comply with https://github.com/intel/llvm/blob/sycl/CONTRIBUTING.md#pull-request, in this case could you please add tags to the beginning of the PR title?

**Comment by FreddyLeaf** [comment]
> > > LGTM. Thanks
> > 
> > 
> > Thanks review! Would you also help merge? I don't have write access.
> 
> @FreddyLeaf, hi, we have the following process for merging: https://github.com/intel/llvm/blob/sycl/CONTRIBUTING.md#merge:
> 
> ```
> ... Feel free to ping @intel/llvm-gatekeepers if your PR is green and can be merged. ...
> ```
> 
> The PR must also comply with https://github.com/intel/llvm/blob/sycl/CONTRIBUTING.md#pull-request, in this case could you please add tags to the beginning of the PR title?

Thanks for the info!

**Comment by FreddyLeaf** [comment]
ping @intel/llvm-gatekeepers, would you help merge pls?

**Pull request merged**